diff --git a/node_modules/react-native-builder-bob/lib/utils/compile.js b/node_modules/react-native-builder-bob/lib/utils/compile.js
index c347764..26be6f6 100644
--- a/node_modules/react-native-builder-bob/lib/utils/compile.js
+++ b/node_modules/react-native-builder-bob/lib/utils/compile.js
@@ -57,7 +57,7 @@ async function compile({
     const result = await babel.transformAsync(content, {
       babelrc: babelrc,
       configFile: configFile,
-      sourceMaps: true,
+      sourceMaps: false,
       filename: filepath,
       ...(babelrc || configFile ? null : {
         presets: [[require.resolve('@babel/preset-env'), {
